core/src/main/kotlin/knote/api/PageManager.kt

```diff
@@ -2,62 +2,16 @@ package knote.api
 
 import knote.script.PageScript
 import knote.util.KObservableMap
+import kotlin.reflect.KType
 import kotlin.script.experimental.api.ScriptDiagnostic
 
 interface PageManager {
     val pages: KObservableMap<String, Page>
```

Comment:
oh wait what are we doing with all that stuff then? I'm using PageManager rn, I'm gonna need to adjust this in the GUI too